README.md

```diff
@@ -1,24 +1,60 @@
 # astronomer-airflow-version-check
 
-An Apache Airflow plugin that will periodically (default, once per day) check
-if there is a new version of Astronomer's Certified Airflow and display a
-message in the UI
+An Apache Airflow plugin that will periodically (default, once per day) check if there is a new version of Astronomer's Certified Airflow and display a message in the UI.
```

Comment:
This line shouldnt show any difference(other than the `fullstop`, I dont see any content change)

Reply:
I think this is because of the indentation change in Readme. I have changed the indentation back to original.